core/service/src/vault/storage/file.rs

```diff
@@ -146,6 +151,30 @@ impl StorageReader for FileStorage {
         }
         Ok(res)
     }
+}
+
+impl StorageReader for FileStorage {
```

Comment:
Should there be some documentation update on the `StorageReader` since it will now fail with non-epoched data?

Reply:
updated docs here 43698d124b844beb8fe4f7b30cf2a6e2ebec40d9